guides/source/active_record_callbacks.md

```diff
@@ -21,6 +20,21 @@ During the normal operation of a Rails application, objects may be created, upda
 
 Callbacks allow you to trigger logic before or after an alteration of an object's state.
 
+An example of this might be when a new user registers for your website, you want to send an welcome email. Active Record makes this easy using callbacks.
+
+```ruby
+class User < ApplicationRecord
+  after_create :send_welcome_email
```

Comment:
Some people argue that callbacks are not a place for sending emails...
https://dhh.dk/2012/emails-are-views.html

Reply:
@lazaronixon I see what you're saying, but I couldn't come up with a better example for now.

If you have one please send a patch!

Reply:
Also one could end up in a rare situation where the delivery job is picked up before the transaction has been committed. I would argue to use ´#after_create_commit´ at least.

Reply:
Thanks for the feedback, but I think the purpose of this example is to simply show the reader what callbacks look like not necessarily to demonstrate best practices (yet).

I've simplified the example in [bf52111](https://github.com/rails/rails/pull/46996/commits/bf52111e274d7c435a75d4fb4830abb21e2e8d7d)